### PR TITLE
[Idle Detection article] Add DevTools support

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add details about DevTools support.

### CODE REVIEW

File: src/site/content/en/blog/idle-detection/index.md [hunk 1]
```diff
@@ -163,6 +163,21 @@ controller.abort();
...
+You can see the various options in the video below.
+
+<figure class="w-figure">
+    <video controls autoplay loop muted class="w-screenshot">      
+      <source src="./devtools.mp4" type="video/mp4">
+    </video>
+   <figcaption class="w-figcaption">
+      Idle Detector state emulation in DevTools.
+    </figcaption>
+  </figure>
```
Comment (tomayac) [inline src/site/content/en/blog/idle-detection/index.md:179]:
```suggestion
<figure class="w-figure">
  <video controls autoplay loop muted class="w-screenshot">
    <source src="./devtools.mp4" type="video/mp4">
  </video>
  <figcaption class="w-figcaption">
    Idle Detector state emulation in DevTools.
  </figcaption>
</figure>
```

**Comment by tomayac** [comment]
@kaycebasques Could you please help with moving the MP4 to the proper storage bucket and then remove the committed file and rewrite the URL?

File: src/site/content/en/blog/idle-detection/index.md [hunk 2]
```diff
@@ -163,6 +163,21 @@ controller.abort();
 console.log('IdleDetector is stopped.');
 ```
 
+### DevTools support
+
+You can emulate idle events in the Chrome DevTools, no need to actually idle around.
```
Comment (mathiasbynens) [inline src/site/content/en/blog/idle-detection/index.md:168]: Worth mentioning that this is Chrome 86+?

File: src/site/content/en/blog/idle-detection/index.md [hunk 3]
```diff
@@ -246,6 +261,7 @@ and let us know where and how you're using it.
 ## Acknowledgements
 
 The Idle Detection API was implemented by [Sam Goto](https://twitter.com/samuelgoto).
+DevTools support was added by [Maksim Sadym](https://www.linkedin.com/in/sadym/).
```
Comment (mathiasbynens) [inline src/site/content/en/blog/idle-detection/index.md:264]: @maximgrafin PTAL

File: src/site/content/en/blog/idle-detection/index.md (hunk 2 above)
Comment (tomayac) [inline src/site/content/en/blog/idle-detection/index.md:168]:
```suggestion
Starting in Chrome&nbsp;86, you can emulate idle events in the Chrome DevTools, no need to actually idle around.
```

File: src/site/content/en/blog/idle-detection/index.md [hunk 4]
```diff
@@ -163,6 +163,21 @@ controller.abort();
 console.log('IdleDetector is stopped.');
 ```
 
+### DevTools support
+
+Starting in Chrome&nbsp;86, you can emulate idle events in the Chrome DevTools, no need to actually idle around.
```
Comment (jpmedley) [inline src/site/content/en/blog/idle-detection/index.md:168]:
```suggestion
Starting in Chrome&nbsp;86, you can emulate idle events in Chrome DevTools without needing to idle around.
```

Comment (jpmedley) [reply src/site/content/en/blog/idle-detection/index.md:168]: What does 'idle around' mean? I get what you're saying. What I'm having trouble with is that 'idle' is a lack of action, yet you're  treating it like an action. I wonder if there's a better way to say that.

Comment (tomayac) [reply src/site/content/en/blog/idle-detection/index.md:168]: "you can emulate idle events in Chrome DevTools without actually being idle."

The reverse were amazing: you can emulate busy events in Chrome DevTools without actually being busy. :-)

File: src/site/content/en/blog/idle-detection/index.md (hunk 2 above)
Comment (tomayac) [reply src/site/content/en/blog/idle-detection/index.md]: Done.

**Pull request merged**